### PR TITLE
DE battle translation adjustments

### PR DESCRIPTION
**Pull request by karl-police**
Matches consistency from Scarlet and Violet ``BTL_STRID_STD_PutSingle_Player``

Somehow the ``!`` in ``playerGo`` reappeared after ``Los``

### CODE REVIEW

**Comment by bennybroseph** [comment]
Do you have a source for this change and a screenshot of how it looks in game?

File: src/locales/de/battle.ts [hunk 1]
```diff
@@ -7,9 +7,9 @@ export const battle: SimpleTranslationEntries = {
   "singleWildAppeared": "Ein wildes {{pokemonName}} erscheint!",
   "multiWildAppeared": "Ein wildes {{pokemonName1}}\nund {{pokemonName2}} erscheinen!",
   "playerComeBack": "Komm zurück, {{pokemonName}}!",
-  "trainerComeBack": "{{trainerName}} ruft {{pokemonName}} zurück!",
-  "playerGo": "Los! {{pokemonName}}!",
-  "trainerGo": "{{trainerName}} sendet {{pokemonName}} raus!",
+  "trainerComeBack": "{{trainerName}} hat {{pokemonName}} zurückgerufen!",
+  "playerGo": "Los, {{pokemonName}}!",
+  "trainerGo": "{{trainerName}} schickt\n{{pokemonName}} in den Kampf!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:12]: {{trainerName}} setzt\n{{pokemonName}} ein!", - This is the text used in Most of Scarlet/Violet (For example the academy tournament

Comment (CodeTappert) [reply src/locales/de/battle.ts:12]: IMG_20240517_173228-min
1715960331126-min

So honestly we can just keep your change! I am fine with that

**Comment by bennybroseph** [comment]
definitely want it to match the in-game text where possible

**Comment by CodeTappert** [comment]
> definitely want it to match the in-game text where possible

All of his changes do that. "trainerGo" Text just has two different strings for in the original game depending on if it is the first pokemon of the trainer or any subsequent.

**Comment by bennybroseph** [comment]
okay, we just need to make that change then

File: src/locales/de/battle.ts (hunk 1 above)
Comment (karl-police) [reply src/locales/de/battle.ts:12]: I looked at the dumped versions, it's a bit hard. The dumper shows interpolation variables as "VAR" without much context.

**Comment by bennybroseph** [comment]
Once the requested changes have been made, feel free to reopen this request

**Comment by karl-police** [comment]
> Once the requested changes have been made, feel free to reopen this request

I can't re-open it don't let me.

**Pull request closed without merging**